_redirects

```diff
@@ -463,3 +463,18 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
 /reference/warehouse-profiles/firebolt-profile /reference/warehouse-setups/firebolt-setup 302
 /reference/warehouse-profiles/mysql-profile /reference/warehouse-setups/mysql-setup 302
 /reference/warehouse-profiles/hive-profile /reference/warehouse-setups/hive-setup 302
+/guides/getting-started/building-your-first-project /docs/develop/getting-started/building-your-first-project/build-your-first-models 301
+/guides/getting-started/building-your-first-project/build-your-first-models /docs/develop/getting-started/building-your-first-project/build-your-first-models 301
+/guides/getting-started/building-your-first-project/schedule-a-job /docs/develop/getting-started/building-your-first-project/schedule-a-job 301
+/guides/getting-started/building-your-first-project/test-and-document-your-project /docs/develop/getting-started/building-your-first-project/test-and-document-your-project 301
+/guides/getting-started/create-a-project /docs/develop/getting-started/create-a-project 301
+/guides/getting-started/getting-set-up /docs/develop/getting-started/getting-set-up 301
+/guides/getting-started/getting-set-up/setting-up-bigquery /docs/develop/getting-started/getting-set-up/setting-up-bigquery 301
+/guides/getting-started/getting-set-up/setting-up-databricks /docs/develop/getting-started/getting-set-up/setting-up-databricks 301
+/guides/getting-started/getting-set-up/setting-up-redshift /docs/develop/getting-started/getting-set-up/setting-up-redshift 301
+/guides/getting-started/getting-set-up/setting-up-snowflake /docs/develop/getting-started/getting-set-up/setting-up-snowflake 301
+/guides/getting-started/getting-started /docs/develop/getting-started 301
+/guides/getting-started/learning-more /docs/develop/getting-started/getting-started-dbt-core 301
```

Comment:
just flagging, is line 477 intentional?

Reply:
Yea if we remove that learning more "Overview" page, I feel like this is the best place to take people.

Reply:
ah i see! makes sense